modules/accounting/includes/classes/AccountingTutorial.php

```diff
@@ -0,0 +1,343 @@
+<?php
+
+namespace WeDevs\ERP\Accounting\Includes\Classes;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * AccountingTutorial Class
+ *
+ * Make a Quick tour tutorial to demonstrate accounting module
+ *
+ * @since 1.8.7
+ */
+class AccountingTutorial {
```

Comment:
Kindly name the class in Snake case, `i.e., Accounting_Tutorial`
Also the file name should be like this. `Accounting_Tutorial.php`